modules/ROOT/pages/_partials/include-traffic-inspection.adoc

```diff
@@ -0,0 +1,90 @@
+
+
+//
+//tag::traffic-inspection-overview[]
+Intercept and inspect all HTTPS traffic through a proxy server that resides between your on-premises Mule apps and Anypoint Monitoring. To enable traffic inspection for your xref:runtime-manager::deployment-strategies.adoc#hybrid-deployments[hybrid deployments], you must install the latest Anypoint Monitoring agent.
+//end::traffic-inspection-overview[]
+//
+//
+//tag::traffic-inspection-limitations[]
+
+At this time, the Anypoint Monitoring agent that supports traffic inspection is available for Mule instances running on _Linux_. 
+//end::traffic-inspection-limitations[]
+//
+//
+//
+//tag::traffic-inspection-prereqs[]
+
+**  A closed beta version of the Anypoint Monitoring agent from a Salesforce representative
++
+The agent's zip file is available to a limited customer base.
+** A Mule server that is registered in Runtime Manager 
```

Comment:
The traffic inspection for the Runtime Management data has been enabled in your standalone mule instances.